apps/bfd-model/bfd-model-rif/pom.xml

```diff
@@ -54,6 +47,38 @@
 			<artifactId>metrics-core</artifactId>
 		</dependency>
 
+		<dependency>
+			<!-- Uses the `dev/rif-layout-and-fhir-mapping.xlsx` file to codegen the
+				model classes in the gov.cms.bfd.model.rif package. -->
+			<groupId>gov.cms.bfd</groupId>
+			<artifactId>bfd-model-codegen</artifactId>
+			<version>${project.version}</version>
+
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+			<!-- The Hibernate JPA Model Generator, which will produce MetaModel classes
+				for JPA entities. This happens happens automagically via javac's annotation
+				processor. -->
+			<groupId>org.hibernate</groupId>
+			<artifactId>hibernate-jpamodelgen</artifactId>
+			<version>${hibernate.version}</version>
+			<exclusions>
+				<exclusion>
+					<groupId>org.jboss.logging</groupId>
+					<artifactId>jboss-logging</artifactId>
+				</exclusion>
+			</exclusions>
+
+			<scope>provided</scope>
+		</dependency>
+			<!-- Used for annotations in the generated code. -->
+		<dependency>
+		<groupId>jakarta.annotation</groupId>
+		<artifactId>jakarta.annotation-api</artifactId>
+		<version>1.3.5</version>
```

Comment:
formatting is off on this block and I would also move this version up into dependencyManagement since it goes along with the other formerly EE provided libs.